### ISSUE TITLE
Overlapping of signature

### ISSUE THREAD

**Issue by swapnilskate** [thomasjbradley/signature-pad#148]
I have an issue using lenevo tablet. I have used multiple signature on my app. For some signature it works fine but for some signature I get border overflow of signature.
![screenshot_2015-03-06-21-41-26](https://cloud.githubusercontent.com/assets/8245439/6549727/ef1ca63a-c652-11e4-9116-82a3e9d2ce11.png)

**Comment by swapnilskate** [comment]
Would you please help me out. I have used your code for my app.
Thank you for this excellent code which is working fine except this bug.
this happens in android device only

**Comment by thomasjbradley** [comment]
That's really unusual. Is there possibly a hidden `canvas` behind the one you're signing on?

**Comment by swapnilskate** [comment]
Hey thanks man for an awesome library. I have resolved this by changing overflow:visible property to each an every parent of canvas. This very issue was reported here and there I saw people suggesting the approach of overflow : visible.

This is a brilliant attempt made by you and open sourcing it so that people like us could be benefited.

The only problem here is now hardware acceleration in my android device.

I am using Rhodes4.0.0 for app development. I have 4 signature pads on one view and that is why if I go on capturing signature the quality goes on decreasing. yo can see from pictures that curves are not smooth and subsequently smoothness disappears. After 6-7 signature attempt of a cirlce drawing results in a square.

Would like to know how can I improve smoothness with hardware acceleration and without hardware acceleration.

Thank You.
